packages/obonode/obojobo-chunks-question/viewer-component.js

```diff
@@ -242,6 +248,12 @@ export default class Question extends React.Component {
 		this.setState({ isFlipping: true })
 		setTimeout(() => this.setState({ isFlipping: false }), DURATION_FLIP_TIME_MS)
 	}
+	//delays refocus on input to account for saved text rendering causing a visual loss of focus
+	onEnterPress() {
+		setTimeout(() => {
+			this.assessmentComponentRef.current.inputRef.current.focus()
+		}, 80)
```

Comment:
This is still kind of a magic number. I'd set it to a constant at the top of the file and use that here instead, like how `DURATION_FLIP_TIME_MS` is used above.

Reply:
Const added. It is still kinda a magic number though. It was the lowest number that reliably delayed refocus enough to retain focus on all browsers.